tools/functional-tester/etcd-tester/tester.go

```diff
@@ -122,20 +128,32 @@ func (tt *tester) runLoop() {
 					return
 				}
 			}
-			if !isSameValueInMap(hashes) {
+			if _, ok = getSameValue(hashes); !ok {
```

Comment:
You mean `currentRevision, ok = getSameValue(revs); ok {`? This is `getSameValue` for hash.